wd_core.au3

```diff
@@ -819,7 +822,7 @@ EndFunc   ;==>_WD_ElementAction
 ;                  $sScript    - Javascript command(s) to run
 ;                  $sArguments - [optional] String of arguments in JSON format
 ;                  $bAsync     - [optional] Perform request asyncronously? Default is False
-;                  $vSubNode  - [optional] Return the designated JSON node instead of the entire JSON string. Default is "" which returns the entire response from web driver
+;                  $vSubNode  - [optional] Return the designated JSON node instead of the entire JSON string. Default is $_WD_JSON_RawResponse an raw response from web driver

```

Comment:
I prefer the prior version (I previously mentioned that adding "raw" wasn't adding any value IMO)

Reply:
I would prospose small change
`Default is $_WD_JSON_RawResponse which returns the entire response in JSON format from web driver`